.github/CODEOWNERS

```diff
@@ -100,6 +100,8 @@
 /cmd/agent/subcommands/integrations     @DataDog/software-integrity-and-trust @DataDog/agent-integrations @DataDog/agent-shared-components
 /cmd/agent/subcommands/remoteconfig     @Datadog/remote-config
 /cmd/agent/subcommands/snmp             @DataDog/network-device-monitoring
+/cmd/agent/subcommands/dogstatsd*       @DataDog/agent-metrics-logs
+/cmd/agent/subcommands/streamlogs       @DataDog/agent-metrics-logs
```

Comment:
nit: not alphabetized in the subcommands grouping

Reply:
Ah, I think that will get shuffled around in the conflicts from landing all of these, anyway.  We can fix it up afterward.

Reply:
I think I got this right in the merge conflict resolution

Reply:
Yup!